nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -103,16 +107,10 @@
             .description("The name of the shared access policy. This policy must have Listen claims.")
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .expressionLanguageSupported(ExpressionLanguageScope.NONE)
-            .required(true)
-            .build();
-    static final PropertyDescriptor POLICY_PRIMARY_KEY = new PropertyDescriptor.Builder()
-            .name("Shared Access Policy Primary Key")
-            .description("The primary key of the shared access policy")
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
-            .sensitive(true)
-            .required(true)
+            .required(false)
             .build();
+    static final PropertyDescriptor POLICY_PRIMARY_KEY =  AzureEventHubUtils.POLICY_PRIMARY_KEY;
+    static final PropertyDescriptor USE_MANANGED_IDENTITY = AzureEventHubUtils.USE_MANAGED_IDENTITY;
```

Comment:
I did not catch this typo earlier: MANANGED here too, and also in the other 2 processor classes.

Reply:
fixed.

Reply:
@sjyang18 I cannot see this fix in your last commit. It is MANANGED in all the 3 classes.

Please also remove the unused getConnectionString() from PutAzureEventHub (with the import).

Reply:
I fixed typos, not sure what went wrong with my IDE not to change in bulk update last time This time, I used grep to double check. 
And, I cannot delete getConnectionString() without impacting existing test case and its BogusConnectionStringMockPutAzureEventHub. So, I decided to move AzureEventHubUtils.getSharedAccessSignatureConnectionString inside getConnectionString.

Reply:
BogusCoonectionStringMockPutAzureEventHub class is defined in PutAzureEventHubTest:351 and its overriding method getConnectionString.

Reply:
I did not notice the BogusCoonectionStringMockPutAzureEventHub test class. Your workaround is fine with me.